### PR TITLE
Add more partners and refactor partners section

### PR DESCRIPTION
**Pull request by okonet**
2018-02-08 at 12 51

### CODE REVIEW

**Comment by okonet** [comment]
I'm having issue with https://github.com/ReactFinland/media/blob/master/logo/v2/logo-black-with-text.svg

@bebraw can you check the size please? See https://deploy-preview-21--reason-conf.netlify.com

File: src/partners.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+"use strict";
```
Comment (okonet) [inline src/partners.js:1]: @ryyppy why is this not in `.gitignore`?

**Comment by bebraw** [comment]
```css
.partners--logo img {
  height: 80px;
}
```

or so to fix.

File: src/partners.js (hunk 1 above)
Comment (ryyppy) [reply src/partners.js:1]: not sure... we have to manually ignore the `.js` files, since we cannot use a `bs.js` prefix in gatsby

**Pull request merged**